addon/helper.js

```diff
@@ -1,15 +1,27 @@
 import Ember from "ember";
 
-export default Ember.Helper.extend({
-  i18n: Ember.inject.service(),
+const { get, inject, Helper, Object: EmberObject, observer } = Ember;
 
-  compute: function(params, interpolations) {
-    const key = params[0];
-    const i18n = this.get('i18n');
-    return i18n.t(key, interpolations);
+function mergedContext(objectContext, hashContext) {
+  return EmberObject.extend({
```

Comment:
I _think_ we're going to run into problems when a property of `objectContext` changes. We'll want a test for
```hbs
{{t 'some.key' obj hashKey=hashValue}}
```
where we change `obj.someProperty` and `hashValue` and make sure both are reflected in the translated text.